src/execution.js

```diff
@@ -1,5 +1,5 @@
 // Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
-import { getData, getDataForVis } from './execution/experimental-executions';
+import { getData, loadAttributesMap } from './execution/experimental-executions';
```

Comment:
[Q] Would you say that `loadAttributesMap` belong to execution?

Let me ask in another way – Would you be looking for `loadAttributesMap` function in such module?

Reply:
moved to separate file